.github/workflows/sycl-post-commit.yml

```diff
@@ -19,6 +19,7 @@ on:
     - ./devops/actions/cleanup
     - ./devops/actions/cached_checkout
 
+
```

Comment:
note: we recommend triggering postcommit using a branch named `sycl-devops-pr` rather than editing the yml file

Reply:
Like opening a PR to `sycl-devops-pr`?

Reply:
I think you name your branch as `sycl-devops-pr`, so like right now it's named `update-free-mem-test` instead you would do `sycl-devops-pr`.

However I have never actually tried this myself to be honest

Reply:
@aelovikov-intel Do you know how it's supposed to work?

Reply:
I guess you'd have to make a branch in `intel/llvm` with `sycl-devops-pr/` as prefix to your branch's name.
For example: https://github.com/intel/llvm/pull/13611